src/utils/object.js

```diff
@@ -36,6 +36,8 @@ export const isObject = obj => obj !== null && typeof obj === 'object'
  */
 export const isPlainObject = obj => Object.prototype.toString.call(obj) === '[object Object]'
 
+export const clone = obj => ({ ...obj })
```

Comment:
Just wondering if this should be called `shallowCopy` or `shallowClone`?

Reply:
Would make it more clear for sure...
We have `cloneDeep()`. `shallowClone()` would fit better IMHO.

Reply:
But on the other hand do we have `cloneDeep()` and `clone()` is nice and short.

Reply:
yeah... or `cloneShallow()` or `cloneNotSoDeep()` :wink:   But clone works as well... maybe just add a comment above the clone definition stating that it is a shallow copy?

Reply:
@tmorehouse Yeah, the comment seems like a good solution.